src/components/PersonLink.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type PeopleLinkProps = {
+  person: Person | null;
+};
+
+export const PersonLink: React.FC<PeopleLinkProps> = ({ person }) => {
+  if (!person) {
+    return <>{'-'}</>;
+  }
+
+  return (
+    <a
+      href={`#/people/${person.slug}`}
+      className={classNames({ 'has-text-danger': person.sex === 'f' })}
```

Comment:
The className logic is correct, but ensure that 'has-text-danger' is defined in your CSS for the expected styling.